### DISCUSSION TITLE
Proxy inside docker container

### DISCUSSION THREAD

**Discussion by Wakotu** [daeuniverse/dae#647] [Q&A] [answered]
I wonder if programs in a docker container could be proxied running dae in host machine? 
It seems not without additional configuration. Is there any way to do so?

**Comment by oluceps** [comment, marked as answer]
https://github.com/daeuniverse/dae/blob/cec5e71d4e99b3a359a88d05317f18e18463b9c7/example.dae#L32

**Reply by Wakotu** [reply]
Thanks for reply, it really works
